scripts/migrate_registration_logs.py

```diff
@@ -0,0 +1,51 @@
+import logging
+
+from modularodm import Q
+
+from website.models import NodeLog
+from website.app import init_app
+
+from scripts import utils as script_utils
+from framework.transactions.context import TokuTransaction
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.INFO)
+
+
+def get_targets():
+    logs = NodeLog.find(
+            Q('action', 'eq', 'embargo_approved') |
+            Q('action', 'eq', 'embargo_initiated') |
+            Q('action', 'eq', 'retraction_approved') |
+            Q('action', 'eq', 'retraction_initiated') |
+            Q('action', 'eq', 'registration_initiated')
+    )
+    return logs
+
+
+def migrate_log(logs):
+    for log in logs:
+        if log.node.registered_from:
+            log.params['node'] = log.node.registered_from_id
```

Comment:
This needs to keep the registered node in the log params.

e.g.
`log.params['registration_node'] = log.params['node']`

@pattisdr is working on a PR